viz/MLSMapVisualization.cpp

```diff
@@ -617,8 +576,61 @@ void MLSMapVisualization::setSimplifySurface(bool enabled)
     setDirty();
 }
 
+int MLSMapVisualization::getMinMeasurements() const
+{
+    return minMeasurements;
+}
+void MLSMapVisualization::setMinMeasurements(int measurements)
+{
+    minMeasurements = measurements;
+    emit propertyChanged("min_measurements");
+    setDirty();
+}
+
+void MLSMapVisualization::visualize(vizkit3d::PatchesGeode& geode, const SurfacePatch<MLSConfig::SLOPE>& p)
+{
+    float minZ, maxZ;
+    p.getRange(minZ, maxZ);
+    minZ -= 5e-4f;
+    maxZ += 5e-4f;
+    Eigen::Vector3f normal = p.getNormal();
+    if(normal.z() < 0)
+        normal *= -1.0;
+    if(p.getNumerOfMeasurements() >= minMeasurements)
```

Comment:
I would suggest to break early `if(p.getNumerOfMeasurements() < minMeasurements) break;` before calculating normals, etc.

Reply:
@chhtz done

Reply:
Thanks!